src/main/java/model/ticket/Ticket.java

```diff
@@ -103,6 +96,15 @@ private void setEventCode(short eventCode) {
         this.eventCode = eventCode;
     }
 
+    @Override
+    public void print() {
+        System.out.println("Class content: " + this);
+    }
+
+    public void share(){
+        System.out.println("Default share");
+    }
+
     @Override
     public String toString() {
```

Comment:
after equals and hashCode

Reply:
Moved method toString() after equals() and hashCode()